extensions/amp-date-display/1.0/date-display.js

```diff
@@ -66,27 +64,25 @@ let EnhancedVariablesV2Def;
  * @return {PreactDef.Renderable}
  */
 export function DateDisplay(props) {
-  const {render, children} = props;
-  const data = /** @type {!JsonObject} */ (getDataForTemplate(props));
+  const {datetime, render, children} = props;
+  const date = getDate(datetime);
+  const data = /** @type {!JsonObject} */ (getDataForTemplate(date, props));
   useResourcesNotify();
 
-  return render(data, children);
+  return data ? render(data, children) : null;
```

Comment:
Nit: I don't really like this overly guarded rendering we're adding. If you pass in incorrect props, I think we should be throwing. In Preact mode, I'd want it to throw. In AMP mode, I'd want it to throw (so resources will blacklist the element).

Reply:
Yeah. I'm trying to figure out good line here. I agree about the `date` in this case, we should just let it fail.

Reply:
Done.